Tests/scripts/validate_files_structure.py

```diff
@@ -252,12 +286,22 @@ def validate_committed_files(branch_name):
                 missing_test = True
                 print_error("You've failed to add the {0} to conf.json".format(file_path))
 
+        if re.match(SCRIPT_REGEX, file_path, re.IGNORECASE) or re.match(INTEGRATION_REGEX, file_path, re.IGNORECASE):
+            id_to_file[get_script_or_integration_id(file_path)] = file_path
+        elif re.match(PLAYBOOK_REGEX, file_path, re.IGNORECASE) or re.match(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE):
+            id_to_file[collect_ids(file_path)] = file_path
+
+    if has_duplicated_ids(id_to_file):
```

Comment:
isn't this already tested in the `validate_all_files`?

Reply:
Yes, but i want to make sure that we do full check on master, just to cover all our bases